fuzzbench/fake_jobs.py

```diff
@@ -15,13 +15,19 @@
 
 import subprocess
 import time
+import typing
 
+from fuzzbench import build_jobs
 
-def build_image(name):
+def build_image(name: str):
     """Build a Docker image."""
     print('Building', name)
-    subprocess.run(['docker', '--version'], check=True)
-    time.sleep(3)
+    if name.startswith('base'):
+        if name in ['base-image', 'base-builder', 'base-runner']:
+            build_jobs.build_base_images(name)
+    else:
+        subprocess.run(['docker', '--version'], check=True)
+        time.sleep(3)
     return True
```

Comment:
Let's just have a jobs.py instead of fake_jobs and build_jobs and use the build_image function currently in build_jobs